src/toast/tests/intervals.py

```diff
@@ -171,6 +171,25 @@ def test_inverse(self):
             included[ival.first : ival.last] += 1
         assert np.all(included == 1)
 
+    def test_closed_time_spans(self):
+        n_samp = 100
+        n_intr = 10
+        stamps = 1000.0 * np.arange(n_samp, dtype=np.float64)
+
+        # Sample ranges are open ended
+        samplespans = [(x * n_intr, x * n_intr + n_intr) for x in range(n_intr)]
+        intr_samples = IntervalList(stamps, samplespans=samplespans)
+
+        # Time ranges are open ended *unless* they fall exactly on a sample.
```

Comment:
time ranges are open-ended, unless the end time coincides with the observation end time.